src/gui/properties/propertieswidget.ui

```diff
@@ -1017,14 +1017,14 @@
          <item>
           <widget class="QPushButton" name="selectAllButton">
            <property name="text">
-            <string>Select All</string>
+            <string>Check all items</string>
```

Comment:
Maybe tick/untick all? Select might imply select entries of list rather than content of these entries, i mean like selecting multiple files in a file manager.